tests/mixed_int8/README.md

```diff
@@ -0,0 +1,13 @@
+# Testing mixed int8 quantization
+
+## Hardware requirements
+
+I am using a setup of 2 GPUs that are NVIDIA-Tesla T4 15GB
+
+## Virutal envs
+
+```conda create --name int8-testing python==3.8```
+```git clone https://github.com/younesbelkada/transformers.git && git checkout integration-8bit```
+```pip install -e ".[dev]"```
+```pip install -i https://test.pypi.org/simple/ bitsandbytes-cuda114```
```

Comment:
I just changed the way bitsandbytes is installed. This line no longer uses the cuda suffix: 
```pip install -i https://test.pypi.org/simple/ bitsandbytes```